wwwroot/cgi-bin/lib/robots.pm

```diff
@@ -2551,8 +2576,8 @@
 'spider','spider',
 'sucker','sucker',
 'validator','validator',
-'bot[\s_+:,\.\;\/\\\-]','Unknown robot identified by bot\*',
-'[\s_+:,\.\;\/\\\-]bot','Unknown robot identified by \*bot',
+'bot[\s_+:,\.;/\\-]','Unknown robot identified by bot\*',
+'[\s_+:,\.;/\\-]bot','Unknown robot identified by \*bot',

```

Comment:
that's a forward slash which is not a special regex character so doesn't need escaping. i.e. without the escape \ then / is treated as / and not as a delimiter.
there may be confusion becasue in some implementations of regex / is used as a start end delimiter but not in this case I think. Infact in PHP which uses PCRE and requires the regex pattern to be enclosed in delimiters, / is often the chosen delimiter. But it is being used as a delimiter in this case and is not a regex special character and so does not need escaping.
Also note when hyphen (-) is the last character inside character class [] it doesn't need to be escaped.  So you only need the \s for whitespace and to escape fullstop and the \ and hence all that is required is:
'[\s_+:,\.;/\\-]bot'

I have been running it like this and it seems to be working fine.